app/models/external/requests/ConsolidationType.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.external.requests
+
+import play.api.libs.json.{Format, Reads, Writes}
+
+object ConsolidationType extends Enumeration {
+  type ConsolidationType = Value
+
+  val ASSOCIATE_DUCR = Value("EAC")
+  val DISASSOCIATE_DUCR = Value("EAC")
+  val SHUT_MUCR = Value("CST")
+
+  implicit val format = Format(Reads.enumNameReads(ConsolidationType), Writes.enumNameWrites)
```

Comment:
I have feeling that this should be not enumName but enumValue or something simillar

Reply:
It building correct Json, it's consistent with backend. Moreover, there is not value now